WalletWasabi.Tests/UnitTests/WabiSabi/Client/ArenaClientTests.cs

```diff
@@ -63,49 +63,50 @@ public async Task FullCoinjoinAsyncTest()
 			// Phase: Input Registration
 			Assert.Equal(Phase.InputRegistration, round.Phase);
 
-			await apiClient.ConfirmConnectionAsync(
+			await aliceArenaClient.ConfirmConnectionAsync(
 				round.Id,
 				aliceId,
 				inputRemainingWeights,
-				amountCredentials.ZeroValue.Take(ArenaClient.ProtocolCredentialNumber),
+				aliceArenaClient.AmountCredentialClient.Credentials.ZeroValue.Take(ArenaClient.ProtocolCredentialNumber),
```

Comment:
Why this? I like it more as it was before, after all `amountCredentials` and `aliceArenaClient.AmountCredentialClient.Credentials` are the same.

Reply:
True, similified.